Dockerfile

```diff
@@ -0,0 +1,52 @@
+# BUILD:
+# docker build --force-rm=true -t openvpn_exporter .
+
+# RUN:
+# docker run -it -p 9176:9176 -v /path/to/openvpn_server.status:/etc/openvpn_exporter/server.status openvpn_exporter
+
+FROM golang as builder
+
+# Grenoble testing env
+ENV http_proxy="http://web-proxy.gre.hpecorp.net:8080" \
+    https_proxy="http://web-proxy.gre.hpecorp.net:8080"
+
+RUN mkdir /app
+RUN mkdir /go/src/app
+ADD . /go/src/app
+WORKDIR /go/src/app
+
+# Go dep
+RUN go get -u github.com/prometheus/client_golang/prometheus && \
+    go get -u github.com/prometheus/client_golang/prometheus/promhttp
```

Comment:
Instead of listing all dependencies explicitly, we could use `go get -d ./...` here.

Reply:
Right, I didn't know this command. I'll update it. Thanks